openfisca_france/model/prestations/education.py

```diff
@@ -322,18 +335,28 @@ class statuts_etablissement_scolaire(Variable):
     definition_period = MONTH
 
 
-class boursier(Variable):
-    value_type = bool
+class TypesClasse(Enum):
+    __order__ = 'autre seconde premiere terminale licence_1 licence_2 licence_3 master_1 master_2 doctorat_1'
+    autre = 'Autre'
+    seconde = 'Seconde'
+    premiere = 'Première'
+    terminale = 'Terminale'
+    licence_1 = 'Licence 1ère année'
+    licence_2 = 'Licence 2ème année'
+    licence_3 = 'Licence 3ème année'
+    master_1 = 'Master 1ère année'
+    master_2 = 'Master 2ère année'
+    doctorat_1 = 'Doctorat 1ème année'
+
+
+class classe_scolarite(Variable):
```

Comment:
Je trouve la notion de « classe » ambiguë et peu appropriée pour les études supérieures. Je comprends que la notion de « niveau d'études » n'est pas approprié non plus puisqu'il fait plus souvent référence à des niveaux de diplômes. Et si on parlait d'[année d'étude](http://gdt.oqlf.gouv.qc.ca/ficheOqlf.aspx?Id_Fiche=26503666), un terme qui me semble revenir dans le supérieur ? 🙂 

```suggestion
class annee_etude(Variable):
```